internal/packages/installer/installer.go

```diff
@@ -25,20 +30,20 @@ type InstalledPackage struct {
 }
 
 // CreateForManifest function creates a new instance of the installer.
-func CreateForManifest(manifest packages.PackageManifest) (*Installer, error) {
+func CreateForManifest(manifest packages.PackageManifest) (Installer, error) {
```

Comment:
Actually this installer doesn't need the whole manifest, right? Only the name and the version.

Reply:
Updated both methods `CreateForManifest` and `CreateForZip` to accept name and version instead of the manifest.